blurring_as_a_service/metrics/upper_blurred_area.py

```diff
@@ -0,0 +1,6 @@
+from metrics.total_blurred_area import TotalBlurredArea
+
+
+class UpperBlurredArea(TotalBlurredArea):
```

Comment:
I like the idea of inheritance in case it adds functionality that the base class doesn't have.
What is the purpose in this case?